app/fighters/knight.py

```diff
@@ -0,0 +1,59 @@
+from app.fighters.tools.poiton import Potion
+from app.fighters.tools.weapon import Weapon
+from app.fighters.tools.armour import Armour
+
+
+class Knight:
+    def __init__(self, name: str, power: int, hp: int,
+                 armour: list[Armour], weapon: Weapon,
+                 potion: Potion | None, protection: int = 0) -> None:
+        self.name = name
+        self.power = power
+        self.hp = hp
+        self.armour = armour
+        self.weapon = weapon
+        self.potion = potion
+        self.protection = protection
+
+    @classmethod
+    def create_knight_objects(cls, knight_config: dict) -> list["Knight"]:
```

Comment:
don't use quotes. if you get error, import annotations from future module. fix everywhere